src/UndoController.js

```diff
@@ -22,45 +28,88 @@ module.exports = class UndoController {
   onKeyPressed(e){
     if((e.ctrlKey ||  e.metaKey) && e.keyCode === 90){
       e.preventDefault();
-      //write the previous content into scribe's element
-      var diff = this.diffs.pop();
-      patch(this.scribe.el, diff);
-
-      //place the caret
-      var selection = new scribe.api.Selection().selection;
-      var markers = this.scribe.el.querySelectorAll('em.scribe-marker');
-      var range = document.createRange();
-      range.collapse(false);
-      selection.removeAllRanges();
-
-      //if there is no selection
-      if(markers.length === 0){
-        return
-      }
-      //if we have a selection
-      else if(markers.length === 2){
```

Comment:
Don't put this in an else. Take advantage of the short-circuit logic from the return